man/bootctl.xml

```diff
@@ -319,11 +319,11 @@
         <term><option>--make-entry-directory=yes|no</option></term>
         <listitem><para>Controls creation and deletion of the <ulink
         url="https://systemd.io/BOOT_LOADER_SPECIFICATION">Boot Loader Specification</ulink> Type #1 entry
-        directory on the file system containing resources such as kernel images and initial RAM disk images
-        during <option>install</option> and <option>remove</option>, respectively. The directory is named
-        after the entry token, as specified with <option>--entry-token=</option> parameter described below,
-        and is placed immediately below the <varname>$BOOT</varname> root directory (i.e. beneath the file
-        system returned by the <option>--print-boot-path</option> option, see above). Defaults to
+        directory on the file system containing resources such as kernel and initrd images during
```

Comment:
btw, adding to the comment in the commit msg, initramfs is a misnomer, as initrd is backed by tmpfs these days, not ramfs. But noone says inittmpfs ;-)

so I am fully on board dropping any references to "initramfs" and "initial RAM disk" and anything like that, and just saying "initrd" everywhere. (Though it might make sense to at the most appropriate place explain what an initrd actually *really* is)

Reply:
Did I say "inittmpfs" somewhere? I don't see it.

There's an explanation in bootup(7). I link to it from one of the man pages in this PR.

Reply:
Oh, OK, I see the other comment. I'll adjust the text.